code/obj/item/device/radio.dm

```diff
@@ -330,6 +330,9 @@ Green Wire: <a href='?src=\ref[src];wires=[WIRE_TRANSMIT]'>[src.wires & WIRE_TRA
 
 	var/list/heard_flock = list() // heard by flockdrones/flockmind
 
+	for (var/obj/item/mechanics/radioscanner/I in connection.devices) //MechComp radio scanner
```

Comment:
I'd probably merge the two loops into one to be more efficient.

Reply:
Ok, this is updated